tests/lib/test_openstack_identity.py

```diff
@@ -198,27 +198,26 @@ def test_delete_project_handles_resource_not_found(self):
         result = self.instance.delete_project("", project_identifier="test")
         assert result is False
 
-    @raises(ValueError)
     def test_delete_immutable_project_throws(self):
         """
         Tests that trying to deleting an immutable project raises an error
         """
         mock_project = self.MockProject(tags=["immutable"])
         self.identity_api.find_project.return_value = mock_project
 
-        self.instance.delete_project(
-            cloud_account="test",
-            project_identifier="ProjectID",
-        )
+        with self.assertRaises(ValueError):
+            self.instance.delete_project(
+                cloud_account="test",
+                project_identifier="ProjectID",
+            )
 
-    @raises(MissingMandatoryParamError)
     def test_find_project_raises_missing_project_identifier(self):
         """
         Tests that a missing mandatory parameter is missing if a whitespace
         or empty string is passed
         """
-
-        self.instance.find_project("set", " ")
+        with self.assertRaises(MissingMandatoryParamError):
+            self.instance.find_project("set", "")
```

Comment:
This used to be passing `" "` - I'm assuming passing `""` is the same - just checking